src/config-schema.coffee

```diff
@@ -155,32 +172,36 @@ module.exports =
       softTabs:
         type: 'boolean'
         default: true
+        description: 'If the `Tab Type` config setting is set to "auto" and autodetection of tab type from buffer content fails, then this config setting determines whether a soft tab or a hard tab will be inserted when the Tab key is pressed.'
```

Comment:
@benogle We could, but is an empty file the only time when this option will be chosen? I think it's not -- it's chosen any time auto-detect fails, and it can fail even when the file is non-empty, as far as I can tell (see my comment above).